wasabi/markdown.py

```diff
@@ -1,36 +1,44 @@
-# coding: utf8
-from __future__ import unicode_literals, print_function
+from typing import List
+
+from typing_extensions import Literal
 
 
 class MarkdownRenderer:
     """Simple helper for generating raw Markdown."""
 
-    def __init__(self, no_emoji=False):
+    def __init__(self, no_emoji: bool = False):
         """Initialize the renderer.
 
         no_emoji (bool): Don't show emoji in titles etc.
         """
-        self.data = []
+        self.data: List = []
         self.no_emoji = no_emoji
 
     @property
-    def text(self):
+    def text(self) -> str:
         """RETURNS (str): The Markdown document."""
         return "\n\n".join(self.data)
 
-    def add(self, content):
+    def add(self, content: str):
         """Add a string to the Markdown document.
 
         content (str): Add content to the document.
         """
         self.data.append(content)
 
-    def table(self, data, header, aligns=None):
+    def table(
+        self,
+        data: List[List[str]],
+        header: List[str],
+        aligns: List[Literal["r", "c", "l"]] = None,
+    ) -> str:
         """Create a Markdown table.
 
-        data (Iterable[Iterable[str]]): The body, one iterable per row,
+        data (List[List[str]]): The body, one iterable per row,
```

Comment:
Why is this changed?

Reply:
Hi @adrianeboyd , I think for the change from Iterable to List, I had a few mypy errors that say something like:

```
wasabi/markdown.py:45: error: Argument 1 to "len" has incompatible type "Iterable[str]"; expected "Sized"
wasabi/markdown.py:46: error: Argument 1 to "len" has incompatible type "Iterable[Literal['r', 'c', 'l']]"; expected "Sized"
wasabi/markdown.py:46: error: Argument 1 to "len" has incompatible type "Iterable[str]"; expected "Sized"
wasabi/markdown.py:47: error: Argument 1 to "len" has incompatible type "Iterable[str]"; expected "Sized"
wasabi/markdown.py:52: error: Value of type "Iterable[Literal['r', 'c', 'l']]" is not indexable
wasabi/markdown.py:52: error: Argument 1 to "len" has incompatible type "Iterable[str]"; expected "Sized"
```

I am not sure what the best type for that is. I searched a bit (cf. [this SO post](https://stackoverflow.com/questions/49427944/typehints-for-sized-iterable-in-python)), and it mentions that a List, Collection or Sequence should work. I opted to use List because we're passing a list internally.